compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/DocumentationAnalyzer.java

```diff
@@ -207,9 +209,11 @@ public void visit(BLangService serviceNode) {
     public void visit(BLangTypeDefinition typeDefinition) {
         BLangType typeNode = typeDefinition.getTypeNode();
         if (typeDefinition.typeNode.getKind() == NodeKind.OBJECT_TYPE) {
-            List<? extends SimpleVariableNode> fields = ((BLangObjectTypeNode) typeNode).getFields();
+            List<BLangSimpleVariable> fields = ((BLangObjectTypeNode) typeNode).fields;
```

Comment:
Why do we need this change?

Reply:
The reason to assign fields into a list of BLangSimpleVariable is that in the method [validateDeprecatedParameter](https://github.com/ballerina-platform/ballerina-lang/blob/4625fbaa458d45ab0b1e8d3984acba4da18b694c/compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/DocumentationAnalyzer.java#L583) we need parameter as BLangSimpleVariable instead of SimpleVariableNode.